addons/google_calendar/models/calendar_recurrence_rule.py

```diff
@@ -84,6 +86,34 @@ def _write_from_google(self, gevent, vals):
 
         base_event_time_fields = ['start', 'stop', 'allday']
         new_event_values = self.env["calendar.event"]._odoo_values(gevent)
+        # We update the attendee status for all events in the recurrence
+        google_attendees = gevent.attendees or []
+        emails = [a.get('email') for a in google_attendees]
```

Comment:
No need to filter in the case where we don't have any email for a google attendee ? 🤔

Reply:
A google attendee is defined by his email. You can't add an attendee on google without a mail, it's mandatory. :-)